Tech@HubAPI/Tech@HubAPI/Controllers/AuthController.cs

```diff
@@ -43,7 +45,11 @@ public IActionResult Login([FromBody] Credentials credentials)
 				return NotFound();
 			}
 
-			// Check password here
+			byte[] hashedPasswordAttempt = _hashing.HashPassword(credentials.Password, user.Salt);
+			if (hashedPasswordAttempt != user.Password)
```

Comment:
So in C# you can't compare two arrays using the `!=` operator, but this should work
```
bool isEqual = Enumerable.SequenceEqual(hashedPasswordAttempt, user.Password);
```